packages/secrets/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 All notable changes to the Zowe Secrets SDK package will be documented in this file.
 
+## Recent Changes
+
+- Enhancement: Added support for secure certificates in keyring for Windows and macOS. [#2325](https://github.com/zowe/zowe-cli/issues/2325)
```

Comment:
We may want to mention here that Linux remains unsupported at this time. It is implied in the error messaging and the code but not explicitly documented, so developers might try to invoke the functions expecting them to work.